### PR TITLE
4611: navigate back to message detail after signing created order

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

File: web-client/src/presenter/actions/completeDocumentSigningActionFactory.js [hunk 1]
```diff
@@ -10,11 +10,9 @@ import { state } from 'cerebral';
...
  * @returns {object} object with new document id
  */
-export const completeDocumentSigningAction = async ({
-  applicationContext,
-  get,
-}) => {
-  const messageId = get(state.currentViewMetadata.messageId);
+export const completeDocumentSigningActionFactory = ({
+  successMessage,
+}) => async ({ applicationContext, get }) => {
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/completeDocumentSigningActionFactory.js]: now the question becomes whether to move the jsdoc to inside the factory, or write new stuff, or leave it as-is? ;P

File: web-client/src/presenter/actions/completeWorkItemForDocumentSigningAction.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { state } from 'cerebral';
+
+/**
+ * Uses state-side signature data (coordinates, page number, PDFJS Object) to apply
+ * the signature to a new PDF and upload to S3, then calls a use case to attach the
+ * new document to the associated case.
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/completeWorkItemForDocumentSigningAction.js]: c/p?

File: web-client/src/presenter/sequences/gotoSignOrderForMessageSequence.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { setCurrentPageAction } from '../actions/setCurrentPageAction';
+import { setParentMessageIdAction } from '../actions/setParentMessageIdAction';
+import { setupSignOrder } from './gotoSignOrderSequence';
+
+export const gotoSignOrderForMessageSequence = [
+  ...setupSignOrder,
```
Comment (kkoskelin) [inline web-client/src/presenter/sequences/gotoSignOrderForMessageSequence.js]: Luckily we do not have to splat-in these sequences. You can drop them in directly and they get flattened for you.

File: web-client/src/presenter/sequences/gotoSignOrderSequence.js [hunk 4]
```diff
@@ -21,5 +20,9 @@ export const gotoSignOrderSequence = [
   setSignatureNameForPdfSigningAction,
   setPDFForSigningAction,
   setPDFPageForSigningAction,
+];
+
+export const gotoSignOrderSequence = [
+  ...setupSignOrder,
```
Comment (kkoskelin) [inline web-client/src/presenter/sequences/gotoSignOrderSequence.js]: nit: splats unnecessary.

File: web-client/src/presenter/actions/skipSigningOrderForMessageAction.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import { state } from 'cerebral';
+
+/**
+ * redirects to the draft documents page
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.get the cerebral get function
+ *
```
Comment (matthopson) [inline web-client/src/presenter/actions/skipSigningOrderForMessageAction.js]: We could probably remove this at some point.

**Review by kkoskelin — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]

**Review by matthopson — APPROVED** [review]

**Pull request merged**